src/paradigma/tremor/feature_extraction.py

```diff
@@ -0,0 +1,429 @@
+        The upper bound of the frequency band in Hz (default: 25)
+        
+    Returns
+    -------
+    pd.Series
+        The frequency of the peak across windows
+    """
+    frequency_peak = []
+    
+    freq_range = freq_vect[0]
+    freq_idx = np.where((freq_range>=fmin) & (freq_range<=fmax))
+    for psd_window in total_psd: 
+        peak_idx = np.argmax(psd_window[freq_idx])
+        freq_peak = freq_range[peak_idx]+fmin
+        frequency_peak.append(freq_peak)
+
+    return frequency_peak
+
+def extract_low_freq_power(
+    total_psd: pd.Series,
+    freq_vect: pd.Series,
+    fmin: float = 0.5,
+    fmax: float = 3,
+    spectral_resolution: float = 0.25
+    ) -> pd.Series:
+
+    """Computes the power in the low frequency power range across windows (for slow arm movement detection).
+    
+    Parameters
+    ----------
+    total_psd: pd.Series
+        The total power spectral density of the gyroscope signal
+    freq_vect: pd.Series
+        Frequency vector corresponding to the power spectral density
+    fmin: float
+        The lower bound of the frequency band in Hz (default: 0.5)
+    fmax: float
+        The upper bound of the frequency band in Hz (default: 3)
+    spectral_resolution: float
+        The spectral resolution of the PSD in Hz (default: 0.25)
+        
+    Returns
+    -------
+    pd.Series
+        The power in the low frequency power range across windows
+    """
+
+    low_freq_power = []
+    
+    freq_range = freq_vect[0]
+    freq_idx = np.where((freq_range>=fmin) & (freq_range<fmax))
+    for psd_window in total_psd: 
+        bandpower = spectral_resolution*np.sum(psd_window[freq_idx])
+        low_freq_power.append(bandpower)
+
+    return low_freq_power
+
+def extract_tremor_power(
+    total_psd: pd.Series,
+    freq_vect: pd.Series,
+    fmin: float = 3,
+    fmax: float = 7,
+    spectral_resolution: float = 0.25
+    ) -> pd.Series:
+
+    """Computes the tremor power (1.25 Hz around the peak within the tremor frequency band)
+    
+    Parameters
+    ----------
+    total_psd: pd.Series
+        The total power spectral density of the gyroscope signal
+    freq_vect: pd.Series
+        Frequency vector corresponding to the power spectral density
+    fmin: float
+        The lower bound of the tremor frequency band in Hz (default: 3)
+    fmax: float
+        The upper bound of the tremor frequency band in Hz (default: 7)
+    spectral_resolution: float
+        The spectral resolution of the PSD in Hz (default: 0.25)
+        
+    Returns
+    -------
+    pd.Series
+        The tremor power across windows
+    """
+
+    tremor_power = []
+    
+    freq_range = freq_vect[0]
+    freq_idx = np.where((freq_range>=fmin) & (freq_range<=fmax))
+    for psd_window in total_psd: 
+        peak_idx = np.argmax(psd_window[freq_idx]) + np.min(freq_idx)
+        left_idx =  np.max([0,int(peak_idx - 0.5/spectral_resolution)])
+        right_idx = int(peak_idx + 0.5/spectral_resolution)
```

Comment:
Voor het overzicht zou je een variabele als _boundary_ of _limit_ of iets dergelijks kunnen creëren met `X = 0.5 / spectral_resolution` zodat het overzichtelijk is. Dan kan je met een # kunnen zeggen waarvoor het is. Klein detail, zit de merge niet in de weg.